pkgpanda/build/__init__.py

```diff
@@ -1128,10 +1169,10 @@ def clean():
                 "'src' directory already exists, did you have a previous build? " +
                 "Currently all builds must be from scratch. Support should be " +
                 "added for re-using a src directory when possible. src={}".format(src_dir))
-        os.mkdir(src_dir)
+        os.makedirs(src_dir)
```

Comment:
Why is replacing `os.mkdir()` with `os.makedirs()` necessary?

Reply:
Why is that change necessary now? What new intermediate directories are we creating?

Reply:
I have reverted.